examples/cli/src/get_market_outcomes.ts

```diff
@@ -0,0 +1,17 @@
+import { getMarketOutcomesByMarket } from "@monaco-protocol/client";
+import { PublicKey } from "@solana/web3.js";
+import { getProgram, logJson, log } from "./utils";
+
+async function marketOutcomes(marketPk: PublicKey){
```

Comment:
no get?

Reply:
I think these are just example functions, they're not actually returning the data, just logging it.

Reply:
yeah, just trying to keep them succinct and distinct from the function they are calling `getMarketOutcomesByMarket`